Makefile

```diff
@@ -34,6 +34,7 @@ INSTALLATION_DIR := $(ROOT_DIR)/installation
 HELM_CHART_DIR := $(INSTALLATION_DIR)/kubernetes/helm
 HELM_OCI_REPOSITORY := ghcr.io/openclarity/charts
 DIST_DIR ?= $(ROOT_DIR)/dist
+BICEP_DIR := $(INSTALLATION_DIR)/azure/vmclarity
```

Comment:
this is already defined in line 383, where the value is `$(INSTALLATION_DIR)/azure`

Reply:
Nice catch! Fixed!